SoulsFormats/SoulsFormats/Formats/MSB/MSBE/PointParam.cs

```diff
@@ -2387,34 +2387,34 @@ private protected override void WriteTypeData(BinaryWriterEx bw)
             }
 
             /// <summary>
-            /// Unknown.
+            /// Affects where torrent can be summoned.
             /// </summary>
-            public class HorseProhibition : Region
+            public class HorseRideOverride : Region
             {
-                private protected override RegionType Type => RegionType.HorseProhibition;
+                private protected override RegionType Type => RegionType.HorseRideOverride;
                 private protected override bool HasTypeData => true;
 
                 /// <summary>
-                /// Unknown.
+                /// 1 = Forbid riding torrent, 2 = Permit riding torrent
                 /// </summary>
-                public int UnkT00 { get; set; }
+                public int OverrideType { get; set; }
```

Comment:
Is this actually an enum with 1 and 2 the only values in the game? If so might be good to make this field an actual enum in case we add enum support to the property editor

Reply:
Done.

Reply:
I already added enum support a while back.